src/Microsoft.Extensions.EnvironmentAbstractions/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "description": "Abstractions for making code that uses file system and environment testable.",
```

Comment:
Put it somewhere that makes it seem like it's not real. Give it a name that makes it seem like it's internal. We're not putting out generic environment abstractions.